src/Expokit.jl

```diff
@@ -14,8 +14,16 @@ const expm! = Base.LinAlg.expm!
 # and http://www.maths.uq.edu.au/expokit
 #
 #
-expmv{T}( t::Number, A, vec::Vector{T};
-			tol::Real=1e-7, m::Int=min(30,size(A,1)), norm=Base.norm) = expmv!(t, A, copy(vec); tol=tol, m=m, norm=norm)
+function expmv{T1,T2}(  t::Number, 
+                        A::SparseMatrixCSC{T1,Int64}, 
```

Comment:
OK, I'll change that.
On Feb 7, 2015 6:15 AM, "acroy" <notifications@github.com> wrote:

> In src/Expokit.jl
> <https://github.com/acroy/Expokit.jl/pull/5#discussion_r24288877>:
>
> > @@ -14,8 +14,16 @@ const expm! = Base.LinAlg.expm!
> >  # and http://www.maths.uq.edu.au/expokit
> >  #
> >  #
> > -expmv{T}( t::Number, A, vec::Vector{T};
> > -			tol::Real=1e-7, m::Int=min(30,size(A,1)), norm=Base.norm) = expmv!(t, A, copy(vec); tol=tol, m=m, norm=norm)
> > +function expmv{T1,T2}(  t::Number,
> > +                        A::SparseMatrixCSC{T1,Int64},
>
> I would prefer to keep A untyped. You can use eltype(A) instead of T1.
> This way expmv will work for any type of A, which supports a small set of
> functions (eltype, norm, A_mul_B!, ...).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/acroy/Expokit.jl/pull/5/files#r24288877>.
>